tern/utils/rootfs.py

```diff
@@ -176,15 +176,22 @@ def extract_tarfile(tar_path, directory_path):
 def prep_rootfs(rootfs_dir):
     '''Mount required filesystems in the rootfs directory'''
     rootfs_path = os.path.abspath(rootfs_dir)
-    try:
-        root_command(mount_proc, os.path.join(rootfs_path, 'proc'))
-        root_command(mount_sys, os.path.join(rootfs_path, 'sys'))
-        root_command(mount_dev, os.path.join(rootfs_path, 'dev'))
-        root_command(host_dns, os.path.join(
-            rootfs_path, constants.resolv_path[1:]))
-    except subprocess.CalledProcessError as error:
-        logger.error("%s", error.output)
-        raise
+    if(
+        os.path.exists(os.path.join(rootfs_path, 'proc')) and
+        os.path.exists(os.path.join(rootfs_path, 'sys')) and
+        os.path.exists(os.path.join(rootfs_path, 'dev'))
+    ):
```

Comment:
We want to create `os.path.join(rootfs_path, 'proc')` etc if it doesn't exist. It would be nice if you could assign the result to a variable and use it below.
```
proc_path = os.path.join(rootfs_path, 'proc')
...
if not os.path.exists(proc_path):
    # make the path
```

Reply:
> 
> 
> Thanks for the PR! I have requested some modifications. If you could test your changes by running `tern -l report -i opensuse/tumbleweed` and see if a report gets created, that would be awesome.
> Also, could you please amend your commit message with a body describing the issue and how you have fixed it. Look at the commit logs for examples :)

Yeah Sure will do it by tomorrow ..